libraries/pbrlib/genglsl/lib/mx_microfacet_specular.glsl

```diff
@@ -78,7 +78,7 @@ float mx_ggx_smith_G2(float NdotL, float NdotV, float alpha)
     float alpha2 = mx_square(alpha);
     float lambdaL = sqrt(alpha2 + (1.0 - alpha2) * mx_square(NdotL));
     float lambdaV = sqrt(alpha2 + (1.0 - alpha2) * mx_square(NdotV));
-    return 2.0 / (lambdaL / NdotL + lambdaV / NdotV);
+    return 2.0 * NdotL * NdotV / (lambdaL * NdotV + lambdaV * NdotL);
```

Comment:
Looks good to me. 

Just a quick question -- do we need to handle divide by 0 ?
`(2.0 * NdotL * NdotV) / max(epsilon, lambdaL * NdotV + lambdaV * NdotL)`